turborepo-tests/integration/turbo.json

```diff
@@ -7,8 +7,12 @@
       "dependsOn": ["^topo"]
     },
     "test": {
-      "dependsOn": ["cli#build", "topo"],
-      "env": ["GO_TAG"]
+      "dependsOn": ["cli#build", "topo", "test:setup"]
```

Comment:
Is the dropping of `"env": ["GO_TAG"]` intentional?

Reply:
I thought it belonged in `cli#build` instead, and now that this dependency exist, that would cover our case 
https://github.com/vercel/turbo/blob/main/cli/turbo.json#L5-L6